### ISSUE TITLE
Add autocomplete=false/new-password to all input fields

### ISSUE THREAD

**Issue by wpsmort** [semperfiwebdesign/all-in-one-seo-pack#3081]
Reported here - https://semperplugins.com/support/troubleshooting-all-in-one-seo-pack-pro/google-analytics-id-defaults-to-wordpress-username-and-cannot-be-overridden/

Several users have reported that the Google Analytics field defaults to their user ID when the field is empty and they click Update Options.  This seems to happen only when using Chrome.

**Comment by wpsmort** [comment]
We've received access to two sites where the users have this problem.  I've tested with both sites but am unable to reproduce the problem.  I've tested with dev sites here and been unable to reproduce this.  

The GA Tracking ID field appears to be the only field affected here, however, it's just a standard text input field so there's nothing that separates this field from any other text field in the UI.  

I'm thinking this is possibly some Chrome issue or something related to a Chrome plugin.

We can try adding `autocomplete="false"` to the input field or use jQuery to prevent autofill.
